pyproject.toml

```diff
@@ -47,6 +47,7 @@ tomli = "^2.0.2"
 langchain-chroma = "^0.1.4"
 langchain-anthropic = "^0.2.4"
 langchain-ollama = "^0.2.0"
+langchain-huggingface = "^0.1.2"
```

Comment:
Make this dependency optional, one way is to have an extra module to use this kind of provider. As you can see in requirements.txt poetry adds a lot of new nvidia library dependencies. At the beginning of this project we had a lot of problems related to platform and cuda/nvidia libraries in python. Also, adding hugginface as a provider is out of scope for this task.